### ISSUE TITLE
databases should always have a default retention policy

### ISSUE THREAD

**Issue by beckettsean** [influxdb/influxdb#3699]
In 0.9.2 it is possible to drop the default RP, meaning all subsequent writes that don't explicitly specify an RP will fail.

```
> use test
Using database test
> show retention policies on test
name	duration	replicaN	default
default	0		1		true
foo	3h25m45.678s	1		false

> drop retention policy "default" on test

> show retention policies on test
name	duration	replicaN	default
foo	3h25m45.678s	1		false

> insert foo,bar=baz value=123
ERR: retention policy not found
```

It is not possible to remove the default setting from a retention policy except by explicitly declaring another policy the default. I think the only way to end up without one is to drop the current default RP.

**Comment by otoolep** [comment]
Should be fixed by #4053

**Comment by otoolep** [comment]
All fixed.
